packages/mask/background/tasks/setup.ts

```diff
@@ -3,3 +3,4 @@ import './setup.hmr.js'
 // NotCancellable tasks here
 import './NotCancellable/PrintBuildFlags.js'
 import './NotCancellable/DatabaseCleanup/DropOldDatabase.js'
+import './pending-tasks.js'
```

Comment:
please make it either NotCancellable (then you should move the file to NotCancellable folder) or Cancellable (you should learn how other cancellable task is written).